swagger.json

```diff
@@ -2574,9 +2574,234 @@
                     }
                 }
             }
+        },
+        "/addresses": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Fetch all addresses/geofences for the organization",
```

Comment:
The term `addresses/geofences` is a bit confusing. Could we simplify this to just be `addresses`, with maybe a description that an address contains either a circle or polygon geofence describing the address entry?

Reply:
I agree that addresses/geofences is a bit confusing, but we use this terminology to refer to addresses in the dashboard so customers will likely be expecting this same terminology in the API. I think we should change this to refer to just addresses but will need to change it everywhere, so I propose we have the API docs match the dashboard for now.

cc @samsarahq/design